sdk/cmake/msvc.cmake

```diff
@@ -29,7 +29,15 @@ endif()
 
 add_definitions(/Dinline=__inline /D__STDC__=1)
 
-add_compile_flags("/X /GR- /EHs-c- /GS- /Zl /W3")
+if(NOT USE_CLANG_CL)
+    add_compile_flags("/X /Zl")
+endif()
+
+add_compile_flags("/GR- /EHs-c- /GS- /W3")
+
+if(USE_CLANG_CL)
+    set(CMAKE_CL_SHOWINCLUDES_PREFIX "Note: including file: ")
```

Comment:
Why, did you debug with /showIncludes and diff?
All changes in this file make sense, but I did not go over all options separately.

Reply:
CMake has a mechanism to determine the prefix used, because of the multi languages possibility, and we miss it with clang-cl so I added it here.

Reply:
clang-cl outputs one less newline than the real cl or something IIRC.